database.js

```diff
@@ -7,53 +7,53 @@ bibtexkey: "He_2017_DG",
 bibtex: '@article{He_2017_DG,\n  Title                    = {A discontinuous {G}alerkin method with a bound preserving limiter for the advection of non-diffusive fields in solid Earth geodynamics },\n  Author                   = {Ying He and Elbridge Gerry Puckett and Magali I. Billen},\n  Journal                  = {Physics of the Earth and Planetary Interiors },\n  Year                     = {2017},\n  Pages                    = {23 - 37},\n  Volume                   = {263},\n  Doi                      = {10.1016/j.pepi.2016.12.001},\n  ISSN                     = {0031-9201},\n  Keywords                 = {Geodynamics},\n  Url                      = {https://doi.org/10.1016/j.pepi.2016.12.001}\n}',
 group: 'dg'},
 entry2:{
+text: "Kronbichler, M., Heister, T., &amp; Bangerth, W. (2012). High accuracy mantle convection simulation through modern numerical methods. Geophysical Journal International, 191(1), 12–29. doi:10.1111/j.1365-246x.2012.05609.x",
```

Comment:
Here there is `&amp;` before the last author and later on `&amp;amp;` is used. Is this intentional?